src/dependencies.jl

```diff
@@ -619,7 +619,8 @@ end
 
 function applicable(dep::LibraryDependency)
     if haskey(dep.properties,:os)
-        if (dep.properties[:os] != OS_NAME && dep.properties[:os] != :Unix) || (dep.properties[:os] == :Unix && !is_unix())
+        if (dep.properties[:os] != (@compat @static is_unix() ? Compat.KERNEL : :NT) 
```

Comment:
wrong way around. `:NT` should not appear in the BinDeps API at all, we only want to compare against `:Windows`